src/camvidlog2/data.py

```
@@ -1,15 +1,49 @@
 from pathlib import Path
-from typing import Iterable
+from typing import Iterable, Literal, Union
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)